### ISSUE TITLE
rule-properties-order, option { unspecified: "bottom" } doesn't work correctly

### ISSUE THREAD

**Issue by vlkosinov** [stylelint/stylelint#469]
Hi, thanks for this great tool!
Looks like, there are a little bug with `rule-properties-order` and [{unspecified: "bottom"}](https://github.com/stylelint/stylelint/blob/master/src/rules/rule-properties-order/README.md#unspecified-topbottomignore) option.

.stylelintrc  file:
```json
{
  "rules": {
    "rule-properties-order": [2, ["color", "background"], { "unspecified": "bottom" }]
  }
}
```

Following code emits incorrect message: `Expected property "left" to come before property "color"`

```css
.class-name {
  left: 0;
  color: pink;
  background: orange;
}
```
And this code doesn't emit anything:

```css
.class-name {
  color: pink;
  left: 0;
  background: orange;
}
```

**Comment by davidtheclark** [comment]
Thanks for the bug report! Fixed in #470.
